hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/client/OzoneBucketStub.java

```diff
@@ -210,7 +211,7 @@ public OzoneDataStreamOutput createMultipartStreamKey(String key,
                                                         int partNumber,
                                                         String uploadID)
       throws IOException {
-    String multipartUploadID = multipartUploadIdMap.get(key);
+    String multipartUploadID = keyToMultipartUpload.get(key).getUploadId();
     if (multipartUploadID == null || !multipartUploadID.equals(uploadID)) {
```

Comment:
Wouldn't `keyToMultipartUpload.get(key) == null` indicate "no such upload?  We'd hit NPE in that case.  I guess `MultipartInfoStub#uploadID` would rarely be `null`.

(`createMultipartKey` seems to have the correct logic.)

Reply:
Thank you for catching it. I have updated it as the same logic as `createMultipartKey`.